WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -30,7 +30,7 @@ + (NSArray *)routes
   @[
     [[FBRoute POST:@"/homescreen"].withoutSession respondWithTarget:self action:@selector(handleHomescreenCommand:)],
     [[FBRoute POST:@"/deactivateApp"] respondWithTarget:self action:@selector(handleDeactivateAppCommand:)],
-    [[FBRoute POST:@"/timeouts/implicit_wait"] respondWithTarget:self action:@selector(handleImplicitWaitCommand:)],
```

Comment:
Could you not remove old endpoint and add TODO: comment to remove it. I need migrate your clients before I can remove it